static_src/components/header.jsx

```diff
@@ -41,15 +41,18 @@ export default class Header extends React.Component {
           <a href="/" className={ this.styler('logo') } title="Home">
             <svg className={ this.styler('logo-img') }>
               <use
-                xlinkHref={ this.getImagePath('logo-dashboard') }>
+                xlinkHref={ this.getImagePath('logo-dashboard') }
```

Comment:
Yeah, also for style, enforced by the eslint config. I'm not crazy about this rule, but that's what airbnb says.